CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Change Log
 
+### 3.0.0
```

Comment:
Lingo suggestions:
- the RSpec \`context\` hook, which is available only since RSpec 3.x
- Use RSpec \`example\` block argument instead of the global \`RSpec.current_example\`. This allows to run tests with the \`async-rspec\` gem